code/index.html

```diff
@@ -1,30 +1,211 @@
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+  <body>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <!-- HERO -->
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <header>
+    <div class="hero-image">
+      <div class="navbar">
+        <img class="company-logo" src="./logo_b365.svg" alt="company logo">
+    </div>
+      <div class="hero-text">
+        <h1 class="pulse">BREATHE</h1>
+        <p>Sign up for a 365 challenge!</p>
+        <a href="#anchor">
+        <button type="button" class="hero-button">Sign up here</button>
+        </a>
       </div>
+    </div>
+    </header>
+
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+     <!-- PREAMBLE & Info -->
+
+     <section class="middleSection">
+
+      <div class="preamble">
+        <h2 class="sectionHeading">Awareness & observation</h2>
+        <p>Meditation is not about trying to turn off your thoughts and emotions. It is about training in awareness and learning to observe your thoughts and feelings without judgment.</p>
+        <div class="contact"> 
+          <img class="jes-img" src="./jes.jpg" alt="jessi-mediterar"/>
+          <div>
+            <p>Don´t hesitate to reach out!</p>
+            <p>Phone <span class="bolder"> 0707394622</span></p>
+            <p>E-mail<span class="bolder"> hemmahosjessi@gmail.com</span></p>
+        </div>
+        </div>
       </div>
+
+        <div class="jes-info">
+          <div class="info-text">
+          <h2 class="sectionHeading">344, going on 365 days in a row</h2>
+          <p>I have meditated for three hundred and forty-four days in a row. Now it's starting to feel like routine. It took time. And I still feel a restlessness and my thoughts wander in a single jumble. But I'm sitting. Still. Breathe me through it. Counting the breaths to help keep focus on the particular breaths. Lets the thoughts come and go. And trying to feel how the body feels. Right now. Notice and release. Not quite. The most difficult thing is probably not to evaluate how well the meditation goes. To not just accept that this is how it works today. I will continue to practice. Welcome to join!</p>
+          </div>
+        </div>
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+
+    <!-- Accordion -->
+
+    <section class="middleSection">
+    <!-- <div class="left"> -->
+    <div class="accordion">
+      <h2 class="sectionHeading">Frequently Asked Questions</h2>
+      <div class="question" id="section1">
+        <h2>How can I find the time to meditate in my busy schedule?</h2>
+        <img class="expand-icon" id="icon" src="plus.svg" alt="plus icon">
```

Comment:
About this plus sign for the questions, I think there is an easier way to do this in css. 
I paste it from my project, maybe it helps to make it easier for next time.

.question:after {
  content: '\02795'; /* Unicode character for "plus" sign (+) */ 

.active:after {
  content: "\2796"; /* Unicode character for "minus" sign (-) */
}

Reply:
Exactly! I tried that out, but obviosly in the wrong way... Since I didn´t get it to work. Now I will try again! The thing is, if you want to customize the icon itself. Then I need to use a svg or a font that handles icons right? 
Thank you for reminding me!